### PR TITLE
Session introductions

### PR DESCRIPTION
**Pull request by kathsherratt**
Adding brief session intros for #144 - more to come so opening as a draft, but could also take reviews/merge in now.

I am now trying harder not to mix these up with typo catching but a few corrections slipped in, sorry for the messy git.

### CODE REVIEW

File: sessions/nowcasting.qmd [hunk 1]
```diff
@@ -2,18 +2,23 @@
...
-
-[Introduction to nowcasting](slides/introduction-to-nowcasting)
-
 ```{r echo = FALSE}
 set.seed(123)
 ```
 
-# Objectives
+## Introduction
+
```
Comment (seabbs) [inline sessions/nowcasting.qmd:10]: This one has a different structure (in terms of introducing the slides) to the others

**Review by seabbs — APPROVED** [review]
This all looks good to me aside from one minor issue.

**Review by sbfnk — COMMENTED** [review]
All looks good to me so far - only noticed sometimes we say "we will" and sometimes "we'll" and now I'm wondering if this is by accident or design.

**Comment by seabbs** [comment]
Is this ready for review/to merge?

**Comment by kathsherratt** [comment]
Yes, just need to add for nowcasting and forecasting sessions but better to merge this now and I'll open a new PR for those (or go ahead, I was just going to move some of the existing text up as there is a lot of intro in the existing content).

**Pull request merged**